addons/mail/static/src/scss/kanban_view.scss

```diff
@@ -14,18 +14,8 @@ $o-kanban-attachement-image-size: 80px;
         .o_kanban_image {
             width: $o-kanban-attachement-image-size;
 
-            + div {
```

Comment:
comment for above: class o_kanban_attachment is always set on the o_kanban_record, so selector line 11 is useless